client/state/ui/guided-tours/selectors.js

```diff
@@ -74,6 +78,12 @@ const getTourFromQuery = createSelector(
 	getInitialQueryArguments
 );
 
+export const isNewUser = state => {
+	const user = getCurrentUser( state );
+	const creation = Date.parse( user.date );
+	return ( Date.now() - creation ) <= moment.duration( 1, 'day' ).asMilliseconds();
```

Comment:
Could we use [`moment.isSame`](http://momentjs.com/docs/#/query/is-same/) here? Like:

```js
const creation = moment( user.date );
return creation.isSame( Date.now(), 'day' );
```